exercises/algorithms/src/test/java/com/cbfacademy/BinarySearchTest.java

```diff
@@ -0,0 +1,77 @@
+package com.cbfacademy;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.cbfacademy.search.BinarySearch;
+import com.cbfacademy.search.Search;
+
+@DisplayName(value = "BinarySearch Exercise")
+public class BinarySearchTest {
+    Search binarySearch;
+    int[] testArray = new int[50];
```

Comment:
Test array can be made several magnitudes larger to better approximate real-world usage